src/main/java/com/jmb/springfactory/service/productionorder/ProductionOrderService.java

```diff
@@ -0,0 +1,37 @@
+package com.jmb.springfactory.service.productionorder;
+
+import java.util.List;
+
+import com.jmb.springfactory.exceptions.NotFoundException;
+import com.jmb.springfactory.exceptions.ServiceLayerException;
+import com.jmb.springfactory.model.dto.ProductionOrderDTO;
+import com.jmb.springfactory.service.GenericService;
+
+public interface ProductionOrderService extends GenericService<ProductionOrderDTO, Integer> {
+
+  /**
+   * Find all order defined into a schedule indentified with param idSchedule
+   * @param idSchedule
+   * @return
+   */
+  public List<ProductionOrderDTO> findAllBySchedule(Integer idSchedule) throws NotFoundException;
+
+  /**
+   * Find the order identified with idOrder into schedule identified with idSchedule
+   * @param idSchedule
+   * @param idOrder
+   * @return
+   * @throws NotFoundException
+   */
+  public ProductionOrderDTO findOneByIdInSchedule(Integer idSchedule, Integer idOrder) throws NotFoundException;
+  
+  /**
+   * Creates a new order and add it in the order list from schedule
+   * @param order
+   * @param idSchedule
+   * @return
+   */
+  public ProductionOrderDTO save(final ProductionOrderDTO order, Integer idSchedule) throws ServiceLayerException, 
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)